src/surge-xt/gui/SurgeGUIEditorHtmlGenerators.cpp

```diff
@@ -652,6 +754,7 @@ code {
                         </table>
                     </div>
                 </div>
+                </div>
```

Comment:
Is this right? It's the same indent level as the one above it.

Reply:
Looks like it's an extra. I removed it.